core/misc/tabledrag.js

```diff
@@ -98,15 +113,10 @@ Backdrop.tableDrag = function (table, tableSettings) {
   // Add a link before the table for users to show or hide weight columns.
   $(table).before($('<a href="#" class="tabledrag-toggle-weight"></a>')
     .attr('title', Backdrop.t('Re-order rows by numerical weight instead of dragging.'))
-    .click(function () {
-      if ($.cookie('Backdrop.tableDrag.showWeight') == 1) {
-        self.hideColumns();
-      }
-      else {
-        self.showColumns();
-      }
-      return false;
-    })
+    .click($.proxy(function (e) {
```

Comment:
I see one possible problem with this: [jQuery.proxy() is deprecated](https://api.jquery.com/jQuery.proxy/)

Reply:
@indigoxela it seems that `.proxy()` was deprecated in jQuery v3.3, whereas we are on 1.x in Backdrop. Besides, it is used in many other .js files in core. Should we leave that to be part of https://github.com/backdrop/backdrop-issues/issues/3106 instead?

Reply:
Fixed with [`fdd72a4` (#3870)](https://github.com/backdrop/backdrop/pull/3870/commits/fdd72a40bc1c4c4bbe1c080b3c35a73d85b09c62)